test/Api.Test/AutoFixture/Attributes/ControllerCustomizeAttribute.cs

```diff
@@ -4,6 +4,11 @@
 
 namespace Bit.Api.Test.AutoFixture.Attributes
 {
+    /// <summary>
+    /// Disables setting of Auto Properties on the Controller to avoid ASP.net initialization errors from a mock environment. Still sets constructor dependencies.
+    /// </summary>
+    /// <param name="controllerType">The Type of the controller to allow autofixture to create</param>
```

Comment:
nit: A `param` comment shouldn't go on the class, this should be on the constructor.

Reply:
Fair enough, that's what I get for blindly copying documentation over from another method